.github/workflows/CompatHelper.yml

```diff
@@ -8,19 +8,28 @@ jobs:
   CompatHelper:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - name: Get Julia compatibility
         id: julia_compat
         # NOTE: this requires a julia compat lower-bound with minor version!
         run : |
           version=$(grep '^julia = ' Project.toml | grep -o '".*"' | cut -d '"' -f2)
           echo "::set-output name=version::$version"
-      - uses: julia-actions/setup-julia@latest
+      - uses: julia-actions/setup-julia@v2
         with:
           version: ${{ steps.julia_compat.outputs.version }}
-      - name: Pkg.add("CompatHelper")
-        run: julia -e 'using Pkg; Pkg.add("CompatHelper")'
-      - name: CompatHelper.main()
+          arch: x64
+          show-versioninfo: true
+      - name: Pkg.add
+        shell: julia --color=yes {0}
+        run: |
+          import Pkg
+          Pkg.add("CompatHelper")
+      - name: CompatHelper.main
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: julia -e 'using CompatHelper; CompatHelper.main()'
+          COMPATHELPER_PRIV: ${{ secrets.DOCUMENTER_KEY }}
```

Comment:
Why this change?

Reply:
If I remember correctly, without it, there was an error about permissions. Let's delete it for now.

Reply:
Albeit, `CompatHelper` has it
https://github.com/JuliaRegistries/CompatHelper.jl/blob/e72bd6b815a69d14ccdb0c85b3589bed2a2af704/.github/workflows/CompatHelper.yml#L44